### PR TITLE
Bring migration guide to docs

### PR DESCRIPTION
**Pull request by zoeyTM**
resolves https://github.com/NomicFoundation/hardhat-ignition/issues/758

### CODE REVIEW

File: docs/src/content/ignition/docs/advanced/_dirinfo.yaml [hunk 1]
```diff
@@ -3,5 +3,6 @@ section-title: Advanced
 order:
   - /execution
   - /deployment-artifacts
+  - /migrating
   - /reconciliation
   - /versioning
```
Comment (kanej) [inline docs/src/content/ignition/docs/advanced/_dirinfo.yaml:8]:
```suggestion
order:
  - /execution
  - /deployment-artifacts
  - /reconciliation
  - /versioning
  - /migrating
```

I think it is its own thing so lets put it at the end.

File: docs/src/content/ignition/docs/advanced/migrating.md [hunk 2]
```diff
@@ -0,0 +1,382 @@
+# Migrating from hardhat-deploy
+
+For several years, the hardhat-deploy community plugin has been a go-to solution for deploying smart contracts within the Hardhat community. Recently, we introduced [Hardhat Ignition](/ignition/docs/getting-started), a declarative system for deploying smart contracts on Ethereum, which aims to pick up the torch from `hardhat-deploy` and expand Hardhat's deployment features.
+
+This guide will walk you through migrating your Hardhat project from `hardhat-deploy` to Hardhat Ignition.
```
Comment (kanej) [inline docs/src/content/ignition/docs/advanced/migrating.md:5]:
```suggestion
This guide will walk you through migrating your Hardhat project to Hardhat Ignition from `hardhat-deploy`, a community plugin for deploying smart contracts used within the Hardhat community.
```

I don't think my suggestion is great, but the blog intro needs reworked. What do you think?

File: docs/src/content/ignition/docs/advanced/migrating.md [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+
+Once converted, tests can be checked in the standard way:
+
+```sh
+npx hardhat test
+```
+
+## Feedback welcome
+
+We would love your feedback if you run into any issues or have any feature requests! Please [open an issue on Github](https://github.com/NomicFoundation/hardhat-ignition/issues/new).
```
Comment (kanej) [inline docs/src/content/ignition/docs/advanced/migrating.md:382]:
```suggestion
```

I think we can drop this

File: docs/src/content/ignition/docs/advanced/migrating.md (hunk 2 above)
Comment (zoeyTM) [reply docs/src/content/ignition/docs/advanced/migrating.md:5]: I think yours is fine, personally

**Review by kanej — APPROVED** [review]
LGTM!

**Pull request merged**